apluslms_roman/backends/docker.py

```diff
@@ -22,14 +25,28 @@ class DockerBackend(Backend):
     @cached_property
     def _client(self):
         env = self.environment.environ
-        kwargs = {}
-        version = env.get('DOCKER_VERSION', None)
-        if version:
-            kwargs['version'] = version
-        timeout = env.get('DOCKER_TIMEOUT', None)
-        if timeout:
-            kwargs['timeout'] = timeout
-        return docker.from_env(environment=env, **kwargs)
+        params = {
+            'base_url': env.get('host'),
+            'version': env.get('version'),
+        }
+        if 'timeout' in env:
+            params['timeout'] = env['timeout']
+
+        # false values: 0, false, '', None
+        # true values: 1, true, "yes", unset
+        tls_verify = bool(env.get('tls_verify', False))
```

Comment:
Based on the logic in the comments, we need to have `True` as default, instead of `False`.